data_fusion/plugins/modules/procflows/data_fusion.py

```diff
@@ -210,7 +210,7 @@ def get_fused_xarray(area_def, fuse_data):
 
     final_product_name = fuse_data["final"]["product_name"]
     final_source_name = fuse_data["final"]["source_name"]
-    final_prod_plugin = products.get_plugin(final_source_name, final_product_name)
+    final_prod_plugin = products.get_plugin(f"{final_source_name}.{final_product_name}")
```

Comment:
Do not merge until CLI has been merged, then this will need to be updated to function with the new CLI
@evrose54